clap_blocks/src/querier.rs

```diff
@@ -229,6 +229,17 @@ pub struct QuerierConfig {
         action
     )]
     pub max_concurrent_queries: usize,
+
+    /// Number of bytes that a single table can provide during a query.
+    ///
+    /// The byte counts are estimated values are merely for self-protection reasons.
```

Comment:
```suggestion
    /// Maximum bytes to scan for a table in a query (estimated).
    ///
    /// If IOx estimates that it will scan more than this many bytes 
    /// in a query, the query will error. This protects against potentially unbounded 
    /// memory growth leading to OOMs in certain pathological queries.
```

The "certain pathological queries" is somewhat aspirational now -- hopefully we'll get to the point where this protection is not needed for almost any query